.github/workflows/build-with-bal-test-graalvm-template.yml

```diff
@@ -75,6 +67,21 @@ jobs:
               run: |
                   perl -pi -e "s/^\s*version=.*/version=${{ env.BALLERINA_LANG_VERSION }}/" gradle.properties
                   ./gradlew build -x check -x test publishToMavenLocal --scan --no-daemon
+                  
+            - name: Set up GraalVM
+              uses: graalvm/setup-graalvm@v1
+              with:
+                  java-version: '17.0.7'
+                  distribution: 'graalvm'
```

Comment:
```suggestion
                  distribution: 'graalvm-community'
```

Reply:
We could use the Oracle GraalVM as default for now. I don't see any issues since the action supports it as default

Reply:
@TharmiganK did we check the licence of oracle?

Reply:
No we have not. For now I'll just use the community edition